app/src/main/java/com/g11x/checklistapp/LocalRepository.java

```diff
@@ -0,0 +1,43 @@
+package com.g11x.checklistapp;
+
+import android.content.ContentProvider;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.net.Uri;
+import android.support.annotation.Nullable;
+
+/** Application data repository. */
+public class LocalRepository extends ContentProvider {
```

Comment:
Had a question that got buried in a previous commit. Just so it doesn't get lost in the shuffle: https://github.com/g11x/checklistapp/pull/7#discussion-diff-74830322